package.json

```diff
@@ -63,6 +63,7 @@
     "apollo-server-express": "^1.2.0",
     "apollo-utilities": "^1.0.3",
     "app-module-path": "^2.2.0",
+    "archiver": "^2.1.1",
```

Comment:
These dependencies are resolved during the build process, and should not be included in the base `package.json`.